arm-network/2016-09-01/swagger/network.json

```diff
@@ -3985,14 +3985,12 @@
             "type": "string",
             "description": "The name of the virtual network gateway."
           },
-          {
-            "name": "parameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/VirtualNetworkGateway"
-            },
-            "description": "Parameters supplied to the Begin Reset Virtual Network Gateway operation through Network resource provider."
+          { 
```

Comment:
And if we need to update the api-version, then this will mean fixing the compute-specific resources in the spec.